src/main.ts

```diff
@@ -29,6 +30,7 @@ async function bootstrap() {
 
   app.enable('trust proxy');
   app.useLogger(logger);
+  app.useGlobalPipes(new ValidationPipe());
```

Comment:
This can be taken off. We have this handled in the `app.module.ts` files

```
useFactory: () =>
        new ValidationPipe({
          whitelist: true,
          forbidNonWhitelisted: true,
        }),
```

Reply:
Done